modules/ROOT/partials/individually-licensed-components/import-from-word-and-export-to-word/import-from-word-and-export-to-word-logs-on-premises.adoc

```diff
@@ -42,7 +42,7 @@ An example log for HTTP transport:
     "transport": "http",
     "statusCode": 200,
     "status": "success",
-    "url": "/v1/convert",
+    "url": "/v2/convert",
```

Comment:
```suggestion
    "url": "/v2/convert/html-docx",
```
This url should be the full slug. Either `/v2/convert/html-docx` or `/v2/convert/docx-html`.

Example log that was received from hitting `http://localhost:8081/v2/convert/html-docx`:

`{"level":30,"time":"2024-07-26T06:48:24.397Z","msg":"Request summary","handler":"convertHtmlToDocx","traceId":"af2bf598-5416-45e9-abcd-8d7406dd5783","data":{"duration":309,"transport":"http","statusCode":200,"environmentId":"unknown","status":"success","url":"/v2/convert/html-docx","method":"POST"},"tags":"metrics"}`